basilica/basilica/__init__.py

```diff
@@ -126,6 +128,9 @@ def embed_images(self, images, model='generic', version='default',
         """
         url = '%s/embed/images/%s/%s' % (self.server, model, version)
         data = ({'img': base64.b64encode(img).decode('utf-8')} for img in images)
+        if (type(images) == list):
+            images = map(self.__resize_image, images)
+            data = ({'img': base64.b64encode(img).decode('utf-8')} for img in images)
```

Comment:
This is also being done on line 130.  I think we should delete lines 130-133 and write:
```
data = ({'img': base64.b64encode(self.__resize_image(img)).decode('utf-8')} for img in images)
```